File: DAM2/DI/Tema3/NazariiJavaFX/src/main/java/org/example/nazariijavafx/model/Producto.java
package org.example.nazariijavafx.model;

public class Producto {

    private int id;
    private String codigo;
    private String nombre;
    private String descripcion;
    private double precio;

    public Producto(String codigo, String nombre, String descripcion, double precio) {
        this.codigo = codigo;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.precio = precio;
    }

    public Producto(int id, String codigo, String nombre, String descripcion, double precio) {
        this.id = id;
        this.codigo = codigo;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.precio = precio;
    }

    public String getCodigo() { return codigo; }
    public String getNombre() { return nombre; }
    public String getDescripcion() { return descripcion; }
    public double getPrecio() { return precio; }

    @Override
    public String toString() {
        return nombre;
    }
}

File: DAM2/Ciber/untitled1/src/model/Producto.kt
package model

class Producto(var id: Int, var precio: Double=10.0,var nombre: String?=null, var descripcion: String?=null) {



    /*constructor(id: Int, precio: Double, descripcion: String?):this (id, precio){
        this.nombre = nombre
    }*/

    /*constructor(id: Int, precio: Double, nombre: String, descripcion:String?):this(id, precio) {
        this.nombre = nombre
        this.descripcion = descripcion
    }*/

    fun mostrarDatos(){
        println("Id=$id")
        println("Precio= $precio")
        println("Nombre= ${nombre ?: "Sin definir"}")
        println("Descripcion= ${descripcion ?: "Sin definir"}")

    }




}